tests/test_pyth_client.py

```diff
@@ -0,0 +1,378 @@
+from typing import Any, Dict, Union, Sequence, List
+import pytest
+import base64
+from pythclient.exceptions import NotLoadedException
+from pythclient.pythaccounts import (
+    _ACCOUNT_HEADER_BYTES, _VERSION_2, PythMappingAccount, PythPriceType, PythProductAccount, PythPriceAccount
+)
+
+from pythclient.pythclient import PythClient, V2_FIRST_MAPPING_ACCOUNT_KEY, V2_PROGRAM_KEY, WatchSession
+from pythclient.solana import (
+    SOLANA_DEVNET_HTTP_ENDPOINT,
+    SOLANA_DEVNET_WS_ENDPOINT,
```

Comment:
You should absolutely not use these in test code just in case the pytest-socket plugin gets used. There is also a global solana_client fixture you can use which just uses example.com as the endpoints. It is designed to fail on purpose. Tests need to be hermetic.